common/src/ess.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+use std::{sync::Arc, time::SystemTime, ops::Deref};
```

Comment:
`rust-fmt` insists: 😄

```suggestion
use std::{ops::Deref, sync::Arc, time::SystemTime};
```